Content.Client/PDA/Ringer/RingtoneMenu.xaml.cs

```diff
@@ -11,7 +11,7 @@ namespace Content.Client.PDA.Ringer
     [GenerateTypedNameReferences]
     public sealed partial class RingtoneMenu : FancyWindow
     {
-        public string[] PreviousNoteInputs = new[] { "A", "A", "A", "A", "A", "A" };
+        public string[] PreviousNoteInputs = new[] { "A", "A", "A", "A", "A", "A", "A" };
```

Comment:
if we're already increasing the number of notes then i don't think we need to keep increasing the ringtone length. 

Having a 6-note password with 8 different notes already gets you to a quarter-million options without making the password more cumbersome to input.

Reply:
i would rather err on the side of not adding more UX friction to it without good cause. If code guessing turns out to be a common issue, we can always have more subtle ways of discouraging it like having a cooldown on setting ringtones.